packages/celotool/src/cmds/account/faucet.ts

```diff
@@ -61,7 +61,7 @@ export const handler = async (argv: FaucetArgv) => {
     const goldAmount = await convertToContractDecimals(argv.gold, goldToken)
     const stableTokenAmount = await convertToContractDecimals(argv.dollars, stableToken)
     console.log(
-      `Fauceting ${goldAmount.toFixed()} Gold and ${stableTokenAmount.toFixed()} StableToken to ${address}`
+      `Fauceting ${argv.gold.toFixed()} Gold and ${argv.dollars.toFixed()} StableToken to ${address}`
```

Comment:
Do we need to call `toFixed` here? Was only doing so that bignumber tostring method didn't use exponential notation